bot.py

```diff
@@ -25,18 +29,34 @@ async def format_embed(self, name, anime=False):
         except kadal.MediaNotFound:
             return
 
-        title = (media.title.get('english')
-                 or media.title.get('romaji')
-                 or media.title.get('native'))
+        title = (media.title.get("english")
+                 or media.title.get("romaji")
+                 or media.title.get("native"))
         desc = "***" + ", ".join(media.genres) + "***\n"
         if media.description is not None:
             desc += media.description[:256 - len(desc)] + f"... [(more)]({media.site_url})"
         # dirty half-fix until i figure something better out
-        desc = desc.replace("<br>", "").replace("<i>", "").replace("</i>", "")
-        footer = re.sub(r'.*\.', '', str(media.format))
-        e = discord.Embed(title=title, description=desc, color=0x4286f4)
+        desc = desc.replace("<br>", "").replace("<bR>", "").replace("<Br>", "").replace("<BR>", "").replace("<i>", "").replace("</i>", "").replace("<I>", "").replace("</I>", "")
+        footer = re.sub(r".*\.", "", str(media.format))
+        title_cover_info = "https://img.anili.st/media/" + str(media.id) 
+
+        def title_cover_info_color(title_cover_info=title_cover_info,temp_file="tmp.jpg"):
+
+            opener = urllib.request.build_opener()
+            opener.addheaders = [("User-agent", "Mozilla/5.0")]
+            urllib.request.install_opener(opener)
+            urllib.request.urlretrieve(title_cover_info, temp_file)
+
+            palette_color = fast_colorthief.get_palette(temp_file, color_count=2, quality=100)
```

Comment:
This looks like it would cause the same problem `urllib` does. A synchronous operation can stall the bot. Try looking into one of two solutions.
- Find an asynchronous package that might work well here
- Run this package through an executor. This option is much worse as it reduces performance considerably.

Additionally, this library seems like a scarcely used fork of an even more dead library. I'm not really against using small time libraries but would be very much happier with a more popular one. Most of these libraries are based on PIL which in itself is synchronous, but is a very good library in general. Feel free to consult me if you're getting stuck here :)